base/memory/ref_counted.h

```diff
@@ -334,16 +374,20 @@ class RefCounted : public subtle::RefCountedBase {
   static constexpr subtle::StartRefCountFromZeroTag kRefCountPreference =
       subtle::kStartRefCountFromZeroTag;
 
+  REPLAY_REF_COUNT_DECLARE_STUB();
```

Comment:
It seems pretty weird to be putting a RecordReplayId() member on every single refcounted object, which almost always returns zero.  This sort of pollution will make it harder to rely on compiler type errors to fix problems in the code.